R/query-webgeom.R

```diff
@@ -21,7 +21,7 @@ setMethod(f = "query",signature("webgeom",'character'),
               }
               url <- sprintf('%s?service=WFS&version=%s&request=GetFeature&typename=%s&propertyname=%s',
                              url(.Object), version(.Object), geom(.Object), .Object@attribute)
-              input_list[['key']] = geom(.Object)
+              input_list[['key']] <- sprintf("%s/*[local-name()='%s']", geom(.Object), .Object@attribute)
```

Comment:
yep, exactly. this `*[local-name()='%s']` allows us to match w/o the namespace prefix. So I am really just being explicit about *which* attribute instead of assuming only one comes back. That assumption was fine before because we specify which propertyName to return, but that other attribute was coming in too for this case.